lib/cc/analyzer/formatters/spinner.rb

```diff
@@ -7,11 +7,13 @@ def initialize(text)
         end
 
         def start
-          @thread = Thread.new do
-            loop do
-              @spinning = true
-              spinner.spin
-              sleep 0.075
+          if $stdout.tty?
+            @thread = Thread.new do
```

Comment:
Does something try to kill this thread later? Is it a problem that `@thread` will be `nil` in the non-tty case?